src/main/java/net/minestom/server/entity/Entity.java

```diff
@@ -1528,18 +1529,18 @@ public boolean isRemoved() {
      *
      * @param delay    the time before removing the entity,
      *                 0 to cancel the removing
-     * @param timeUnit the unit of the delay
+     * @param temporalUnit the unit of the delay
      */
-    public void scheduleRemove(long delay, @NotNull TimeUnit timeUnit) {
+    public void scheduleRemove(long delay, @NotNull TemporalUnit temporalUnit) {
```

Comment:
I think if we're moving to use more `java.time` stuff, methods like this where we pass a number and a time unit should just accept `Duratiom` instead.

Reply:
Fair point, i'll deprecate these and add methods that take `Duration`